hellomama_registration/settings.py

```diff
@@ -178,4 +178,14 @@
 CELERY_RESULT_SERIALIZER = 'json'
 CELERY_ACCEPT_CONTENT = ['json']
 
+PREBIRTH_MIN_WEEKS = 10
```

Comment:
these also need `os` options. But wrap them in `int()` because envvars are strings.